src/inputs/SelectField.mock.tsx

```diff
@@ -45,8 +46,10 @@ export function SelectField<T extends object, V extends Key>(props: SelectFieldP
     >
       <option disabled value=""></option>
       {options.map((option, i) => {
+        const isDisabled =
+          disabledOptions && disabledOptions.includes(getOptionValue(option).toString()) ? true : false;
```

Comment:
I was thinking you wouldn't need a ternary statement here as `isDisabled` would return `undefined | Boolean`, but maybe we can't use `undefined` as a value of `disabled`? If you use the suggestion of defaulting to an empty array, then you could omit the `disabledOptions && ` part of the conditional, then it'll always return a Boolean, at which point the `? true : false` is redundant.

Reply:
yeah you are right, it didn't like `undefined` as a value for `disabled` so put the ternary in there but yeah can remove it now with the other approach. Will make this update - thanks!